src/core/Akka.MultiNodeTestRunner.TrxReporter/Models/UnitTest.cs

```diff
@@ -0,0 +1,57 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="UnitTest.cs" company="Akka.NET Project">
+//      Copyright (C) 2009-2019 Lightbend Inc. <http://www.lightbend.com>
+//      Copyright (C) 2013-2019 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+namespace Akka.MultiNodeTestRunner.AzureDevOps.Models
+{
+    using System.Collections.Generic;
+    using System.Xml.Linq;
+    using static XmlHelper;
+
+    public class UnitTest : ITestEntity
+    {
+        public UnitTest(string className, string name, Identifier testListId, string storage)
+        {
+            ClassName = className;
+            Name = name;
+            TestListId = testListId;
+            Storage = storage;
+        }
+
+        public Identifier Id { get; } = Identifier.Create();
+        public Identifier TestListId { get; }
+        public Identifier ExecutionId { get; } = Identifier.Create();
+        
+        public string Name { get; }
+        public string Storage { get; }
+        public string AdapterTypeName => "executor://xunit/MulitNodeTestRunner";
+        public string ClassName { get; }
+
+        public List<UnitTestResult> Results = new List<UnitTestResult>();
```

Comment:
So I think I understand this - these are all of the sub-results for each individual node's test.

I.e. in a multi-node test with 4 nodes, we would expect 5 items in this collection: 1 for each node and 1 for the summary of the results. Is that correct? Just want to make sure I understand how it works.

Reply:
`UnitTest` is a test container. In usual cases, there will be just one `UnitTestResult`, but when, for example, data-driven tests are executed, there could be more results.

Each `UnitTestResult` could have inner test results and each inner test result could have its own inner results too... as much as you need it.

I implemented that each multi-node test will have one `UnitTest` with one `UnitTestResult`, but it will container inner results for each individual node.

```
Multi-node test (UnitTest)
|
+-- Overall test outcome, a summary of all individual nodes (UnitTestResult)
     |
     +-- Node 1 test result (UnitTestResult in InnerResults)
     +-- Node 2 test result (UnitTestResult in InnerResults)
     +-- Node n test result (UnitTestResult in InnerResults)
```

Reply:
Got it, that makes sense